CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+[1.9.1] - 2022-08-05
+--------------------
+
+### Bug Fixes
+
+- argument_validator: fix IPRouteUtils.get_route_tables_mapping() for whitespace
+  sequence
+
+### Other Changes
+
+- network_state: improve state comparison for achieving idempotency
```

Comment:
I think the network state change is a bug fix, and the argument validator is something for the Other category

Reply:
But according to what this issue states, https://github.com/linux-system-roles/network/issues/507, "argument_validator: fix IPRouteUtils.get_route_tables_mapping() for whitespace  sequence" is more like a bugfix.

Reply:
ok - so two bug fixes then, and `- none` for New Features and Other Changes